app/javascript/dashboard/routes/dashboard/settings/attributes/AddAttribute.vue

```diff
@@ -53,21 +54,25 @@
               {{ $t('ATTRIBUTES_MGMT.ADD.FORM.TYPE.ERROR') }}
             </span>
           </label>
-          <div v-if="displayName" class="medium-12 columns">
-            <label>
-              {{ $t('ATTRIBUTES_MGMT.ADD.FORM.KEY.LABEL') }}
-              <i class="ion-help" />
-            </label>
-            <p class="key-value text-truncate">
-              {{ attributeKey }}
-            </p>
-          </div>
+          <woot-input
+            v-model="attributeKey"
+            :label="$t('ATTRIBUTES_MGMT.ADD.FORM.KEY.LABEL')"
+            type="text"
+            :class="{ error: $v.attributeKey.$error }"
+            :error="
+              $v.attributeKey.$error
+                ? $t('ATTRIBUTES_MGMT.ADD.FORM.KEY.ERROR')
+                 : ''
+            "
+            :placeholder="$t('ATTRIBUTES_MGMT.ADD.FORM.KEY.PLACEHOLDER')"
+            @blur="$v.attributeKey.$touch"
+          />
           <div class="modal-footer">
             <woot-submit-button
               :disabled="
                 $v.displayName.$invalid ||
-                  $v.description.$invalid ||
-                  uiFlags.isCreating
+                $v.description.$invalid ||
```

Comment:
We could write a computed prop for this check

Reply:
Done